src/components/Header/HeaderTests/__snapshots__/ToolbarToggle.test.js.snap

```diff
@@ -9,7 +9,7 @@ exports[`ToolbarToggle should render correctly 1`] = `
   data-popper-escaped="true"
   data-popper-placement="bottom-end"
   data-popper-reference-hidden="true"
-  style="position: absolute; left: 0px; top: 0px; z-index: 9999; min-width: 0px; transform: translate(0px, 0px);"
+  style="position: absolute; left: 0px; top: 0px; z-index: 9999; opacity: 1; transition: opacity 0ms cubic-bezier(.54, 1.5, .38, 1.11); min-width: 0px; transform: translate(0px, 0px);"
```

Comment:
FWIW these changes are unrelated to the Jotai change - the version of PF updated with the update to package-lock.json